### DISCUSSION TITLE
Skin Editor doesn't display properly

### DISCUSSION THREAD

**Discussion by Boo-BooLazer** [ppy/osu#22576] [Q&A] [answered]
Skin displays differently when playing through replay vs. actual gameplay
![image](https://user-images.githubusercontent.com/94070063/217751416-503c61bb-6895-4cc7-80c3-487634dc41c5.png)
This image is being played in a replay and it shows at the bottom the color hit error collides with the song progress.
![image](https://user-images.githubusercontent.com/94070063/217751612-d3fd267d-d000-4f2c-8d1f-e84e0f8f5942.png)
This second one is in gameplay, with nofail on and you can see that the color hit error object does not collide. 
I also noticed that changes made with the skin editor in lazer don't accurately translate over to gameplay properly when you edit the skin through the "gameplay" tab in the editor. Not sure if this is an isolated incident.

**Comment by peppy** [comment, marked as answer]
This is because the progress bar has a seek bar you can use to adjust it when watching a replay. Your best bet is to move the other elements slightly higher.

**Reply by Boo-BooLazer** [reply]
haha, i missed the most obvious thing. thanks for the answer
